src/meds_torch/configs/model/backbone/eic_transformer_decoder.yaml

```diff
@@ -1,4 +1,5 @@
 defaults:
+  - generation_budget: seq_len
```

Comment:
I wonder if there is a way to do this that doesn't inject it into the generic decoder config. I'm not sure it is a problem if it is here, but it is worth a (brief) thought.

Reply:
I think I can move this (in a later commit to be at the eic model level. as I can add a _target_ in the eic model yaml which will call the czsl (clinical zero-shot labeler) library, which handles stop criterias and creating labels.